src/panel_gwalker/_gwalker.js

```diff
@@ -112,9 +148,12 @@ export function render({ model }) {
     ref={graphicWalkerRef}
     data={transformedData}
     fields={fields}
+    chart={transformedSpec}
     computation={computation}
     appearance={appearance}
     vizThemeConfig={theme}
+    /* hack to force re-render if the transformedSpec is reset to null */
```

Comment:
We also need a hack to force a re-render if the same spec is sent from backend. For example if the user manually changed the chart(s) and then wants to re-apply the spec in the `spec` parameter.

You can see the problem in action below.

https://github.com/user-attachments/assets/4d089f79-6d3b-4538-9d73-b863c176cc7d